fhir-flow/src/main/java/com/ibm/fhir/flow/app/Main.java

```diff
@@ -132,11 +138,11 @@ public void parseArgs(String[] args) {
                     throw new IllegalArgumentException("missing value for --reader-pool-size");
                 }
                 break;
-            case "--change-id-marker":
+            case "--from-checkpoint":
                 if (i < args.length + 1) {
-                    this.changeIdMarker = Long.parseLong(args[++i]);
+                    this.startFromCheckpoint = args[++i];
                 } else {
-                    throw new IllegalArgumentException("missing value for --change-id-marker");
+                    throw new IllegalArgumentException("missing value for --checkpoint");
```

Comment:
```suggestion
                    throw new IllegalArgumentException("missing value for --from-checkpoint");
```

Reply:
yeah, saw that too when I looked at the diffs! fixed